onos_ric_sdk_py/sdl.py

```diff
@@ -74,6 +75,96 @@ async def get_cell_ids(self, e2_node_id: str) -> List[str]:
         except GRPCError as e:
             raise ClientRuntimeError() from e
 
+    async def _get_cell_entity_id(self, e2_node_id: str, cell_id: str) -> int:
+        """
+        given e2_node_id and cell_id, returns entity id
+        raises ClientRuntimeError if cell_id is not found
+        """
+        if not self._ready:
+            raise ClientStoppedError()
+
+        client = TopoStub(self._topo_channel)
+        filters = Filters(
+            relation_filter=RelationFilter(
+                src_id=e2_node_id,
+                relation_kind=RanRelationKinds.CONTAINS.name.lower(),
+                target_kind="",
+            )
+        )
```

Comment:
Is there a way to create a filter that can include the `cell_id` information? This way, we wouldn't need to iterate over a list of all cells.

Reply:
no, because I don't think the information stored within the aspects are queryable

Reply:
True. Presently, we don't have a query that can reach into the contents of an aspect. However, information about the cell should be directly obtainable from the topo if you know the cell ID. Subtract the PLMNID (via masking operation) and convert the number to hex string representation (with lowercase a-f) and you have an ID for the topo entity that corresponds to the E2 cell (should have e2cell aspect, among others possibly)